docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -393,18 +396,19 @@ quarkus.keycloak.devservices.resource-aliases.policies=/policies.jar <1>
 # Map the policies archive to a specific location in the container
 quarkus.keycloak.devservices.resource-mappings.policies=/opt/keycloak/providers/policies.jar <2>
 ----
-<1> Creates a `policies` alias for the `/policies.jar` classpath resource. The policies archive can also be located on the file system.
+<1> Creates a `policies` alias for the `/policies.jar` classpath resource.
+The policies archive can also be located on the file system.
 <2> Maps the policies archive to the `/opt/keycloak/providers/policies.jar` location inside the Keycloak container.
 
 == Running the application in JVM mode
 
-After exploring the application in dev mode, you can run it as a standard Java application.
+After exploring the application in dev mode, you can run it as a standard Java application in JVM mode.
 
-First compile it:
+Compile the application:
 
 include::{includes}/devtools/build.adoc[]
 
-Then run it:
+Run the application in JVM mode:
```

Comment:
This section is about running in JVM mode, and it was further clarified just above

Reply:
Updated. Thanks.